### ISSUE TITLE
SDK - No connection case

### ISSUE THREAD

**Issue by mawlle** [stagecast/bug-report#133]
**Describe the bug**
Lets inform the user why it's nothing is loaded if they have no connection and try to launch the SDK.

**To Reproduce**
1. Launch SDK from parent app when the is no connection
This screen remains until the app crashes:
![img_4129](https://user-images.githubusercontent.com/18073152/51105048-388b1000-17e8-11e9-9faf-dcebe36d30d0.PNG)

**Expected behavior**
We could have a simple message saying:

"Det verkar vara problem att ansluta till internet. Kontrollera internetanslutningen och starta om appen"
"It seems to be a problem with the internet connection. Check the conncetion and restart the app"

**Additional context**
We could do something very simple here. Stefan / Filippo -you decide what is feasible and just do that without design from Robin.

**Comment by peffis** [comment]
We can just show an Alert when this happens I guess.

**Comment by filippoboiani** [comment]
> We can just show an Alert when this happens I guess.

Yes, we can either show an alert or a view with a reload button.

**Comment by peffis** [comment]
The question is if this should be done for all requests though and implementing a fetch with a timeout possibly everywhere (it does not have it now). If it's not done on all requests it becomes hard to identify which request is the initial one because "it depends". So I think this should be done for all requests and the correct method would be to present a "reload" button. Given that this could be triggered anywhere "reload" would probably need to mean - restart SDK page. Not sure how one would do that at the moment but this is probably possible. 

So it would need a fetch with a timeout, something like
```javascript
function fetchWithTimeout( url, timeout ) {
    return new Promise( (resolve, reject) => {
        // Set timeout timer
        let timer = setTimeout(
            () => reject( new Error('Request timed out') ),
            timeout
        );

        fetch( url ).then(
            response => resolve( response ),
            err => reject( err )
        ).finally( () => clearTimeout(timer) );
    })
}
```

And then a "reload" for when this timeout occurs.

**Comment by peffis** [comment]
Fixed. Will be included in build 0.0.10
